modules/installation-requirements-user-infra.adoc

```diff
@@ -72,8 +72,7 @@ these cluster machines.
 endif::ibm-z[]
 ====
 
-The bootstrap, control plane, and compute machines must use the {op-system-first} as the
-operating system.
+The bootstrap and control plane machines must use {op-system-first} as the operating system. Compute machines can use {op-system} or {op-system-base-full} 7.8 - 7.9.
```

Comment:
It might be technically accurate, but I don't know if we want to encourage the use of something we've noted as deprecated: https://docs.openshift.com/container-platform/4.6/release_notes/ocp-4-6-release-notes.html#ocp-4-6-byo-rhel-7-deprecated

Reply:
@ashcrow perhaps we just strip the second sentence and just state this?
>The bootstrap and control plane machines must use {op-system-first} as the operating system.

Reply:
:+1: That works!

Reply:
Although that leads to the natural follow-up question of "and what about compute machines?"

The supported RHCOS/RHEL versions are listed just a bit below this information in a table, so I don't think it's all that critical to be super detailed with this statement.